packages/patterns/src/getColorFromGradientStyle.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * Internal dependencies
+ */
+import createSolidFromString from './createSolidFromString';
+import { PatternType, type Gradient, type Linear } from './types';
+
+export const DEFAULT_GRADIENT: Linear = {
```

Comment:
Let's move this to its own `constants.ts` file.

Reply:
Done!